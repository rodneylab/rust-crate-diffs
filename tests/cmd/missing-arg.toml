bin.name = "rust-crate-diffs"
status.code = 2
