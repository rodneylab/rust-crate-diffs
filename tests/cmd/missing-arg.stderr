error: the following required arguments were not provided:
  <REPO_PATH>

Usage: rust-crate-diffs <REPO_PATH>

For more information, try '--help'.
